tests/test_elements/test_mol2any/test_mol2concatenated.py

```diff
@@ -131,6 +137,74 @@ def test_n_features(self) -> None:
             net_charge_elem[1].n_features + 16 + physchem_elem[1].n_features,
         )
 
+    def test_features_names(self) -> None:  # pylint: disable-msg=too-many-locals
+        """Test getting the names of features in the concatenated vector."""
+
+        physchem_elem = (
+            "RDKitPhysChem",
+            MolToRDKitPhysChem(),
+        )
+        net_charge_elem = ("NetCharge", MolToNetCharge())
+        morgan_elem = (
+            "MorganFP",
+            MolToMorganFP(n_bits=16),
+        )
```

Comment:
also add 
```python
 morgan_elem = (
            "MorganFP",
            MolToMorganFP(n_bits=16, use_features=True),
        )
```

Reply:
is this pushed?

Reply:
Ah, this is done [here](https://github.com/basf/MolPipeline/pull/106/files#diff-7e74236900a11b6c8c132dc98b0844d2d1b5e0926371b52bc361fa933e36d413R284)